.gitignore

```diff
@@ -5,3 +5,5 @@
 **/*.zip
 
 !**/.vscode/tasks.json
+sw/deployer/.idea/
+.idea/

```

Comment:
Use `**/.idea` instead of creating specific entries. Please keep the style consistent by moving it above `!..` entry, and don't forget to sort entries alphabetically.